paddle/fluid/platform/mkldnn_reuse.h

```diff
@@ -950,6 +950,74 @@ class TransposeMKLDNNHandler : public MKLDNNHandler {
   std::vector<int> logical_axis_;
 };
 
+template <typename T>
+class AXPYMKLDNNHandler : public platform::MKLDNNHandlerT<T, dnnl::reorder> {
+ public:
+  AXPYMKLDNNHandler(const MKLDNNDeviceContext& dev_ctx,
```

Comment:
Hi, As I understand, this AXPYMKLDNNHandler is defined but never used and tested? There should be a sum mkldnn kernel or AXPY mkldnn kernel?

The test_sum_op is testing CPU naive sum op not mkldnn sum op right?

Reply:
I have removed this handler as it's not used.